### PR TITLE
#111 Add ESLINT rule for consistent object curlybar spacing

### LINKED ISSUE #111: Add ESLINT rule for consistent object curlybar spacing
Use the `object-curly-spacing` lint rule to keep consistent spacing before and after the curly braces in object literals. See discussion here: https://github.com/zalando/tailor/pull/109/files/e118752324243635d0e0cf453c979966f8bff83c#diff-72eec7d9bc03ee5b16aa5768cb53d614

### PR DESCRIPTION
**Pull request by addi90**
- [x] Add ESLINT rule `object-curly-spacing`
- [x] Fix all existing issues
- Closes #111 

Refer: https://github.com/jlongster/prettier rule: `"prettier.bracketSpacing": true`

### CODE REVIEW

**Comment by dazld** [comment]
looks good, but what's wrong with the saucelabs token in the tests?

File: .eslintrc [hunk 1]
```diff
@@ -20,6 +20,7 @@
         "no-multi-spaces": 2,
         "no-unused-vars": 2,
         "no-lonely-if": 2,
+        "object-curly-spacing": ["error", "always"],
```
Comment (vigneshshanmugam) [inline .eslintrc]: can you make it 2 instead of error?

**Comment by addi90** [comment]
@dazld It doesn't work for non-members :(

**Comment by vigneshshanmugam** [comment]
@addi90 resolved the merge conflicts.. will merge it once test completes

**Pull request merged**